src/main/java/core/basesyntax/Car.java

```diff
@@ -58,14 +58,16 @@ public void setWeight(Integer weight) {
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
         Car car = (Car) o;
-        return Objects.equals(model, car.model) &&
-                Objects.equals(engine, car.engine) &&
-                Objects.equals(color, car.color) &&
-                autoType == car.autoType &&
-                Objects.equals(weight, car.weight);
+        return Objects.equals(model, car.model) && Objects.equals(engine, car.engine)
+                && Objects.equals(color, car.color) && autoType == car.autoType
```

Comment:
why not Objects.equals for autoType?

Reply:
this is enum, and this equals was generated by Idea